src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -181,7 +177,7 @@ protected virtual HashAlgorithmName GetHashAlgorithmName(string algorithm)
             throw LogHelper.LogException<ArgumentOutOfRangeException>(LogMessages.IDX10640, algorithm);
         }
 
-        private void ResolveDotNetCoreAsymmetricAlgorithm(AsymmetricSecurityKey key, string algorithm, bool willCreateSignatures)
+        public virtual void ResolveAsymmetricAlgorithm(AsymmetricSecurityKey key, string algorithm, bool willCreateSignatures)
```

Comment:
Hmm, I'm not sure to understand what's the point of making `ResolveAsymmetricAlgorithm` public and virtual if it still returns void and if the fields are all private :fearful: